sdk/src/main/java/io/dapr/client/domain/DaprBulkAppResponse.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2021 The Dapr Authors
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package io.dapr.client.domain;
+
+import java.util.Arrays;
+
+/**
+ * Response from the application containing status for each entry in the bulk message.
+ * It is posted by the bulk subscribe handler.
+ */
+public class DaprBulkAppResponse {
+  private DaprBulkAppResponseEntry[] statuses;
+
+  /**
+   * Instantiate a DaprBulkAppResponse.
+   */
+  public DaprBulkAppResponse() {
+  }
+
+  /**
+   * Instantiate a DaprBulkAppResponse.
+   * @param statuses array of statuses.
+   */
+  public DaprBulkAppResponse(DaprBulkAppResponseEntry[] statuses) {
```

Comment:
Is it possible to use List here ?

Reply:
Kept it as an array to not force users to use `List` and keep the contract primitive. Users are still free to use a `List` and use `myList.toArray()` to create a `DaprBulkAppResponse`.